src/manager.ts

```diff
@@ -14,11 +14,11 @@ import { StatusBarNotifier } from './server/serverUtils/statusBarNotifier';
 import { LIVE_PREVIEW_SERVER_ON } from './utils/constants';
 import { ServerGrouping } from './server/serverGrouping';
 import { UpdateListener } from './updateListener';
+import { URL } from 'url';
```

Comment:
I recommend always using vscode's URI instead of node's. It will get confusing to mix both

Reply:
I use URL here because I need to parse out the port number